lib/widgets/profile.dart

```diff
@@ -33,6 +35,36 @@ class ProfilePage extends StatelessWidget {
       page: ProfilePage(userId: userId));
   }
 
+  /// The given user's real email address, if known, for displaying in the UI.
+  ///
+  /// Returns null if [selfUser] isn't able to see [user]'s real email address.
```

Comment:
`selfUser` is formatted here as though it's an identifier in the function's code, but it isn't actually. So let's just say "the self-user".

(Here and below.)